Makefile.inc

```diff
@@ -0,0 +1,5 @@
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
+GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
```

Comment:
I'd rather leave these two in the root `Makefile` until another `Makefile` needs them, but there is some appeal to keeping all the `GIT_*` together.  Personally, my preference for quick `make` calls outweighs my desire to group `GIT_*`, but I can't speak for the maintainers.

Reply:
yes I was also not sure how to split this file, and I preferred to leave all similar variables in the same place.  Thanks for the input, I'll move them to the main Makefile